lib/actions/ui.js

```diff
@@ -174,9 +174,9 @@ export function matchContentToUrl(location) {
       case 'route':
         if (id) {
           // This is a bit of a hack to check if the route details have been grabbed
-          // agencyName will only be populated if route details are repsent
+          // bikesAllowed will only be populated if route details are repsent
```

Comment:
Typo "~repsent~" => "present"

Reply:
Not sure about `bikesAllowed` either - that field is not populated by many Atlanta agencies.

Reply:
Would the presence of at least one pattern be a more reliable criterion?

Reply:
Good catch thank you! and `bikesAllowed` feels strong to me because even if it's not populated OTP still sets it to 0 which allows us to do this null check

Reply:
One pattern was what I initially had, but ran into problems where sometimes routes wouldn't have patterns (?) although the full route was downloaded.

Reply:
Typo fixed in a500cbe779f68c61c9e9739c30b640641fdb2b1a